modules/pruning-images-manual.adoc

```diff
@@ -0,0 +1,325 @@
+`--certificate-authority` is specified:
+
+. The `prune` command is run with the `--force-insecure` option.
+. The provided `registry-url` is prefixed with the `http://` scheme.
+. The provided `registry-url` is a local-link address or `localhost`.
+. The configuration of the current user allows for an insecure connection. This
+can be caused by the user either logging in using `--insecure-skip-tls-verify`
+or choosing the insecure connection when prompted.
+
+[IMPORTANT]
+====
+If the registry is secured by a certificate authority different from the one
+used by {product-title}, it must be specified using the
+`--certificate-authority` flag. Otherwise, the `prune` command fails with an
+error.
+====
+
+[id="pruning-images-problems_{context}"]
+== Image pruning problems
+
+[discrete]
+[id="pruning-images-not-being-pruned_{context}"]
+==== Images not being pruned
+
+If your images keep accumulating and the `prune` command removes just a small
+portion of what you expect, ensure that you understand the image prune
+conditions that must apply for an image to be considered a candidate for
+pruning.
+
+Ensure that images you want removed occur at higher positions in each tag
+history than your chosen tag revisions threshold. For example, consider an old
+and obsolete image named `sha:abz`. By running the following command in
+namespace `N`, where the image is tagged, the image is tagged three times in a
+single imagestream named `myapp`:
+
+----
+$ image_name="sha:abz"
+$ oc get is -n N -o go-template='{{range $isi, $is := .items}}{{range $ti, $tag := $is.status.tags}}'\
+  '{{range $ii, $item := $tag.items}}{{if eq $item.image "'"${image_name}"\
+  $'"}}{{$is.metadata.name}}:{{$tag.tag}} at position {{$ii}} out of {{len $tag.items}}\n'\
+  '{{end}}{{end}}{{end}}{{end}}'
+myapp:v2 at position 4 out of 5
+myapp:v2.1 at position 2 out of 2
+myapp:v2.1-may-2016 at position 0 out of 1
+----
+
+When default options are used, the image is never pruned because it occurs at
+position `0` in a history of `myapp:v2.1-may-2016` tag. For an image to be
+considered for pruning, the administrator must either:
+
+* Specify `--keep-tag-revisions=0` with the `oc adm prune images` command.
++
+[CAUTION]
+====
+This action effectively removes all the tags from all the namespaces with
+underlying images, unless they are younger or they are referenced by objects
+younger than the specified threshold.
+====
+
+* Delete all the `istags` where the position is below the revision threshold,
+which means `myapp:v2.1` and `myapp:v2.1-may-2016`.
+
+* Move the image further in the history, either by running new Builds pushing to
+the same `istag`, or by tagging other image. Unfortunately, this is not always
+desirable for old release tags.
+
+Tags having a date or time of a particular image's Build in their names should
+be avoided, unless the image must be preserved for an undefined amount of time.
+Such tags tend to have just one image in its history, which effectively prevents
```

Comment:
s/Such tags tend to have just one image in its history, which effectively prevents
them from ever being pruned./Such tags tend to have just one image in their history, which prevents
them from ever being pruned./